### ISSUE TITLE
Audio gets cut and freezes the game

### ISSUE THREAD

**Issue by DarioMelo** [TheFakeMateo/RedPlusPlus#30]
When starting a battle with a wild Nidoran F/M the music and their cries sound broken, and then the game freezes before I get any message.

![nidobug](https://user-images.githubusercontent.com/30637436/33532194-4498c156-d876-11e7-84fa-fb8982df5f6e.png)

**Comment by TheFakeMateo** [comment]
Which emulator were you using? And were you trying to use an old save file?

I haven't had a chance to test myself, been afk most of the day, but Danny can't replicate it, nidorans work fine. Since the crysaudio code uses sram like it was wram for now, an old save might be part of your problem. But not sure.

**Comment by DarioMelo** [comment]
I deleted my save but the bug persists. I'm using BGB.

I noticed that the game doesn't completely freeze, if you wait enough (a lot) eventually the battle continues normally but without music. It also kinda happens the same with other Pkmn like Rattata and Mankey, but it freezes only for a few seconds and music plays normally. Another thing is that the wave channel is muted when a battle begins and only starts playing after a sound effect (like a Pkmn cry or a "trainer battle starts" sfx).
